docs/industrial_optimization/productizing_apps.rst

```diff
@@ -0,0 +1,171 @@
+
+    *   Interpreting and converting the problem results into a form suitable for
+        the appropriate business system and users.
+
+    *   Sending notifications to business systems and users.
+    
+A real-world example of an enterprise-level hybrid application is
+Pattison Food Group's
+`production application <https://www.dwavequantum.com/resources/application/e-comm-driver-auto-scheduling-pattison-food-group>`_,
+which uses D-Wave's hybrid solvers to optimize the weekly assignment
+of drivers' work assignments, taking into account various constraints, such as
+seniority and the required number of drivers for each shift.
+
+.. _quantum_apps_guidelines:
+
+Application Guidelines
+======================
+
+An application should be capable of handling large-scale, long-running
+hybrid problems in a heterogeneous environment typical of many enterprise-level
+businesses.
+
+.. _quantum_apps_performance_scaling:
+
+Performance and Scaling
+-----------------------
+
+Some performance and scaling considerations are the following:
+
+*   Desired throughput (i.e., solved problems per second). Also take into
+    account concurrency limits to provision your workloads effectively.
+
+*   Internet connection latency and bandwidth.
+
+*   Size of problems, which can range from about 10 MB to greater than 1 GB.
+
+*   Long and asynchronous jobs; hybrid problems can be configured to run for a
+    few seconds or several hours.
+
+*   CPU and memory requirements for hardware and virtual compute: How you model
+    your problems can impact these requirements.
+
+You should also keep up to date with Ocean SDK releases to help ensure the best
+performance and take advantage of the newest features.
+
+.. _quantum_apps_security:
+
+Security
+--------
+
+In addition to implementing best practices for security and access control,
+you should also consider the following best practices specific to interacting
+with D-Wave's compute infrastructure:
+
+*   Use a minimum-permission model for allowing application access to
+    sensitive data, such as personally identifiable information (PII).
+
+*   Ensure that access control to the application for submitting problems is
+    limited to authorized users.
+
+*   Rotate your solver API token on a regular basis, and change them immediately
+    before deploying to production, just in case they have been saved outside of
+    your codebase during development. In addition, ensure that your solver API
+    token is secure as follows:
+
+    *   Store your solver API token in a single secure location.
+    
+    *   Do not hardcode your solver API token in your application.
+
+    *   Do not write your solver API token to logs nor save it to your version
+        control system.
+
+.. _quantum_apps_monitoring_logging:
+
+Monitoring and Logging
+----------------------
+
+The application should log metrics, including problem IDs, timestamps,
+source, and timing information. A problem ID uniquely identifies each problem
```

Comment:
"timestamps, source, and timing information" what's the difference between timestamps and timing info? And by "source" do you mean filename in the application?